src/base/RadioCheckboxBase.js

```diff
@@ -88,40 +85,35 @@ class RadioCheckboxBase extends Component {
     const { isDisabled, isSelected } = nextProps
 
     if (this.props.isSelected !== isSelected || this.props.isDisabled !== isDisabled) {
-      this.setState({isDisabled, isSelected: isDisabled ? false : isSelected})
+      this.setState({isSelected: isDisabled ? false : isSelected})
     }
   }
 
-  onClick = (event) => {
-    const { btnType, onClick } = this.props
+  handleChange = (event) => {
+    const { btnType, onChange } = this.props
     const { isSelected } = this.state
 
     if (btnType === 'radio' && isSelected) { return }
 
     this.setState({isSelected: !isSelected})
-    onClick(event, {...this.props, isSelected: !isSelected})
+    onChange(event, {...this.props, isSelected: !isSelected})
   }
 
   renderInputBtn() {
-    let bkgImage
-    const { aria, assets, btnType, isDisabled, id, style, value } = this.props
+    let BkgSvg
+    const { aria, bkgSvg, btnType, isDisabled, id, style, value } = this.props
     const { isSelected } = this.state
 
-    if (isDisabled) { bkgImage = assets.disabled }
-    else { bkgImage = isSelected ? assets.checked : assets.base }
+    if (isDisabled) { BkgSvg = bkgSvg.disabled }
+    else { BkgSvg = isSelected ? bkgSvg.selected : bkgSvg.base }
 
     return (
       <div style={{...STYLE.button, ...style.button}}>
-        <img
-          style={STYLE.image}
-          src={bkgImage}
-          tabIndex='-1'
-          alt=''
-        />
+        <BkgSvg />
         <input
           id={id}
           type={btnType}
-          onClick={this.onClick}
+          onChange={this.handleChange}
```

Comment:
The reason for this prop change is that React kept complaining that when setting the `checked` attribute on the `input` tag, an `onChange` callback also had to be defined.